src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -426,7 +425,7 @@ void testParentQueryResult() throws Exception {
   }
 
   @Test
-  void testRejectRequest() {
+  void testRejectSnapshotAccessRequest() {
```

Comment:
I think this is a lost merge update from `develop` and should be reverted
```suggestion
  void testRejectRequest() {
```